plugins/audit_log/lib/audit_log/samson_plugin.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+require 'splunk_logger'
+
+module SamsonAuditLog
+  class Engine < Rails::Engine
+  end
+
+  class Audit
+    class << self
+      VALID_METHODS = [:info, :warn, :debug, :error]
+
+      def plugin_enabled?
+        not (ENV['SPLUNK_TOKEN'].nil? || ENV['SPLUNK_URL'].nil?)
```

Comment:
`ENV['SPLUNK_TOKEN'] && ENV['SPLUNK_URL']` ...
but these might be used for any kind of feature, so a seperate flag for this plugin would be nice ...